src/wp-includes/block-bindings/sources/pattern.php

```diff
@@ -0,0 +1,23 @@
+<?php
+/**
+ * The "pattern" source for the Block Bindings API. This source is used by the
+ * Partially Synced Patterns.
+ *
+ * @since 6.5.0
+ * @package WordPress
+ */
+function pattern_source_callback( $source_attrs, $block_instance, $attribute_name ) {
+	if ( ! _wp_array_get( $block_instance->attributes, array( 'metadata', 'id' ), false ) ) {
+		return null;
+	}
+	$block_id = $block_instance->attributes['metadata']['id'];
+	return _wp_array_get( $block_instance->context, array( 'pattern/overrides', $block_id, $attribute_name ), null );
+}
+
+register_block_bindings_source(
+	'core/pattern-attributes',
```

Comment:
Just a heads-up that we'd have to change it to `core/pattern-overrides`.

Reply:
Thanks! Updated it in 69ed7541a89a432704b52cae3b17b7c520418355